client.go

```diff
@@ -1408,6 +1408,25 @@ func linkProperty(key string, value interface{}) LinkOption {
 	}
 }
 
+// LinkSourceName sets the name of the source queue
```

Comment:
I'd like to see some detail added to this documentation, specifically:
* The link name must be unique on this connection.
* The default will be randomly generated.
* Sentences should end with punctuation. (https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences)

For example:
```go
// LinkName sets the name of the link.
//
// The link names must be unique per-connection.
//
// Default: randomly generated.
``` 

Details about link names: http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-transport-v1.0-os.html#doc-idp298752